eq-author/src/App/page/Preview/QuestionPagePreview.test.js

```diff
@@ -79,7 +79,7 @@ describe("QuestionPagePreview", () => {
 
   it("should render", async () => {
     const { getByText } = render(
-      <MeContext.Provider value={me}>
+      <MeContext.Provider value={{ me: user }}>
```

Comment:
Sorry, I know I probably said it should be user when we went through this. But change these back to `me`. This file should disappear from this PR then.